WordPress/Sniffs/Security/EscapeOutputSniff.php

```diff
@@ -414,7 +414,7 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 				return $end;
 			}
 
-			return $this->check_code_is_escaped( $message_param['start'], ( $message_param['end'] + 1 ) );
+			return $this->check_code_is_escaped( $message_param['start'], ( $message_param['end'] + 1 ), 'ErrorNotEscaped' );
```

Comment:
If you don't want to change this yet, since `trigger_error` having `OutputNotEscaped` is a long-standing thing versus `throw` being new in 3.0.0,
```suggestion
			return $this->check_code_is_escaped( $message_param['start'], ( $message_param['end'] + 1 ), 'OutputNotEscaped' );
```